src/components/data/data-table-provenance.js

```diff
@@ -38,3 +38,63 @@ export const DataTableColumnProvenance = ({ workspace, column, provenance }) =>
     ])
   ])
 }
+
+export const FileProvenance = ({ workspace, fileUrl }) => {
```

Comment:
@nawatts are you thinking of adding a unit test for this?

Reply:
Added tests for this component in https://github.com/DataBiosphere/terra-ui/pull/3353/commits/d0d175c0e04588d7afa7d31546bef5868b13477c, which also revealed a bug (both the workflow and submission links were labeled with the submission ID).